packages/components/src/components/map-marker/map-marker.stories.ts

```diff
@@ -0,0 +1,177 @@
+import '../../solid-components';
```

Comment:
@saemik94 looks good but I am missing the migration guide. Pls add it to the guides collection.

Reply:
I added the migration guide :)